packages/edge-worker/src/EdgeWorker.ts

```diff
@@ -1154,29 +1221,37 @@ export class EdgeWorker extends EventEmitter {
 
 	/**
 	 * Handle agent session created webhook
-	 * Can happen due to being 'delegated' or @ mentioned in a new thread
-	 * @param webhook The agent session created webhook
-	 * @param repos All available repositories for routing
+	 * . Can happen due to being 'delegated' or @ mentioned in a new thread
+	 * @param webhook
+	 * @param repos All configured repositories
 	 */
 	private async handleAgentSessionCreatedWebhook(
-		webhook: LinearAgentSessionCreatedWebhook,
+		webhook: AgentSessionCreatedWebhook,
 		repos: RepositoryConfig[],
 	): Promise<void> {
-		const issueId = webhook.agentSession?.issue?.id;
+		const { agentSession, guidance } = webhook;
+		const linearAgentActivitySessionId = agentSession.id;
```

Comment:
dont call it `linearAgentActivitySessionId` that defeats our attempt to remove "linear" from this file /code